exonum-java-binding-core/rust/integration_tests/tests/service_proxy.rs

```diff
@@ -194,3 +210,90 @@ fn service_can_modify_db_on_initialize() {
         .expect("Failed to find the entry created in the test service");
     assert_eq!(INITIAL_ENTRY_VALUE, value);
 }
+
+#[test]
+#[should_panic(expected = "Java exception: com.exonum.binding.fakes.mocks.TestException")]
+fn after_commit_throwing() {
+    let service = ServiceMockBuilder::new(EXECUTOR.clone())
+        .state_hashes(&[hash(&[1, 2, 3])])
```

Comment:
I wouldn't consider that a drawback :-) 

The usual approach with the test builders is to make the builder provide the defaults, so that the client code does not have to specify parameters that are not relevant to that test case. Can we make the ServiceMockBuilder always set some 32-byte hash?

FYI, more on that in this blog post: https://testing.googleblog.com/2018/02/testing-on-toilet-cleanly-create-test.html
and the book, which I highly recommend.